tier2/checklist.md

```diff
@@ -0,0 +1,413 @@
+| Business Owner(s)       | BUSINESS OWNER 1 </n> BUSINESS OWNER 2   | [Approved/Needs Approval] </n> [Approved/Needs Approval]     |
+
+
+
+
+### Flipping the Switch: Making the Repository Public
+Once the repository has passed outbound review, we are ready to “flip the switch” and
+officially make it public. Once made public, there are a couple of actions that need to be
+taken:
+
+#### Repository Actions
+
+Please enable the following features to enhance repository security and maintain code
+quality:
+
+- [ ] Dependabot Alerts
+
+    *A GitHub Feature. Get notified when one of your dependencies has a vulnerability*
+
+- [ ] Secret Scanning Alerts
+
+    *A GitHub Feature. Get notified when a secret is pushed to this repository. Ideally set this up to run after each new commit is pushed to the Repository.*
+
+- [ ] Branch Protections
+
+    *Ensures the integrity of important branches by preventing unauthorized actions like force pushes and requiring pull request reviews with specific checks before merging. Dev and main should be protected branches in the repository.*
+
+- [ ] Git Branching
+
+    *After making the repository public, make sure there is a coherent git branching plan in place. For example: agree to merge feature related pull requests into dev but merge bug fixes into main instead of dev first.*
+
+- [ ] Add Repolinter GH Action to CI
+
+    *For ongoing adherence to repository hygiene standards, integrate the [repolinter GitHub Action](https://github.com/DSACMS/metrics/blob/main/.github/workflows/checks.yml) into your CI pipeline. This addition enhances your workflow by automatically enforcing repository cleanliness standards.*
+
+- [ ] Optional: DCO (Developer Certificate of Origin)
+
+    *Requires all commit messages to contain the Signed-off-by line with an email address that matches the commit author. The Developer Certificate of Origin (DCO) is a lightweight way for contributors to certify that they wrote or otherwise have the right to submit the code they are contributing to the project. The GitHub app to enforce DCO can be found [here](https://github.com/apps/dco) .*
+
+
+#### Communications & Rollout 📣
+
+Share the good news with communities both inside and outside CMS!
+
+- [ ] Draft a launch announcement
+
+    *Be sure to include the following information:*
+ - Repo Description
+ - Repo URL
+ - Authoring Team Email Contact
+ - Authoring Team URL
+ - Authorting Team Slack Channel
+ - Call to action (file issues, contribute PRs)
+- [ ] Post launch announcement to CMS slack channels
+ - #cms-opensource
+ - #cms-api-community
+ - #cms-data-community
+ - #cms-engineering-community
+ - #ai-community
+- [ ] Send a launch announcement email
+- [ ] Add launch announcement to a Confluence Wiki Page
+
+#### Tracking 📈
+
+Add your project to our inventories.
+
+- [ ] Add to https://github.com/dsacms/open projects inventory
+- [ ] Add code.json to repository and sent over a pull request to [code.gov](code.gov)
```

Comment:
This link doesn't work for me. I think it should work if you changed it to `https://code.gov` instead of just `code.gov` in the parenthesis.